src/components/Book.js

```diff
@@ -1,28 +1,31 @@
 import React from 'react';
 import propTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/Books';
+import { removeBookStore } from '../redux/books/Books';
 
-const Book = ({ id, title, author }) => {
+const Book = ({ book }) => {
+  const { title, author } = book;
   const dispatch = useDispatch();
 
-  const clickHandler = () => {
-    dispatch(removeBook(id));
+  const clickHandler = (id) => {
+    dispatch(removeBookStore(id));
   };
 
   return (
     <li>
       <p>{title}</p>
       <p>{author}</p>
-      <button type="submit" onClick={clickHandler}>Remove</button>
+      <button type="button" onClick={() => clickHandler(book.item_id)}>Remove</button>
```

Comment:
## Please  remove book features should work the same as before even after the refactoring to use the API has been done
![Capture](https://user-images.githubusercontent.com/63972670/192045151-b10f0e9d-f477-4e1d-8e52-c92798e551a4.PNG)

Reply:
Thank you for pointing that out, i applied the changes